### PR TITLE
feat: migrate to dumi-theme-antv from gatsby

### PR DESCRIPTION
**Pull request by LAI-X**
### AVA Website Migration

image

### CODE REVIEW

**Review by hustcc — APPROVED** [review]

File: sites/ava-site/docs/api/ckb/CKBOptions.en.md [hunk 1]
```diff
@@ -25,7 +24,7 @@ CKBOptions(lang)
 
 *object* * contains the following keys:
 
-## `CKBOptions().family`
+##### `CKBOptions().family`
```
Comment (chenluli) [inline sites/ava-site/docs/api/ckb/CKBOptions.en.md:27]: why change the `##`  to heading `#####` in this doc?

File: packages/lite-insight/src/visualization/index.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const getInsightVisualizationSchema = (
   const { dimensions, patterns, measures } = insight;
 
   const schemas: VisualizationSchema[] = [];
-  const summaryType = _get(visualizationOptions, 'summaryType', 'text');
+  const summaryType: any = _get(visualizationOptions, 'summaryType', 'text');
```
Comment (chenluli) [inline packages/lite-insight/src/visualization/index.ts:34]: @BBSQQ this type can be specified?

File: sites/ava-site/docs/api/ckb/CKBOptions.en.md (hunk 1 above)
Comment (chenluli) [reply sites/ava-site/docs/api/ckb/CKBOptions.en.md:27]: done

Comment (LAI-X) [reply sites/ava-site/docs/api/ckb/CKBOptions.en.md:27]: after migration, the font has become larger

File: packages/lite-insight/src/visualization/index.ts (hunk 2 above)
Comment (BBSQQ) [reply packages/lite-insight/src/visualization/index.ts:34]: will optimize it in v3 later.

File: sites/ava-site/.dumirc.ts [hunk 3]
```diff
@@ -27,7 +27,14 @@ export default defineConfig({
     // 是否在 demo 页展示API文档 
     showAPIDoc: true,   
     // 是否展示国内镜像链接     
-    showChinaMirror: true,                           
+    showChinaMirror: true,
```
Comment (hustcc) [inline sites/ava-site/.dumirc.ts:30]: 这个配置不需要。只需要下面的

Comment (LAI-X) [reply sites/ava-site/.dumirc.ts]: 已修改

**Review by hustcc — APPROVED** [review]

**Pull request merged**